src/domain/reservation/manage/list/ManageReservationsList.js

```diff
@@ -79,6 +81,7 @@ ManageReservationsList.propTypes = {
   reservations: PropTypes.array,
   units: PropTypes.array,
   intl: intlShape,
+  toggleModal: PropTypes.func
```

Comment:
Same here, `toggleModal` is not a good name for this callback.

Reply:
Renamed